uiowa/uiowa
# Grad aos updates

Resolves #4682 



# How to test
`yarn workspace uids_base build`
`blt ds --site=grad.admissions.uiowa.edu`
- Review the Files changed
- Go to https://admissionsgrad.uiowa.ddev.site/academics/accounting-mac and verify that:
  - Sidebar is reorganized according to #4682 
  - Subtitle field has been removed
  - Page title is no longer all caps
  - New apply button link field as been added and should appear as part of the "Apply" box in the sidebar
  - Domestic costs link shows up as a button
  - Each text area has space between it and the `block__margin-bottom` class is applied to each block in the text area. 

The page should look like this:

![Screenshot 2022-01-21 at 15-35-38 Accounting (MAc)](https://user-images.githubusercontent.com/1036433/150603268-5dfa99e5-c86d-4f92-969e-22176fcf1730.png)
